frontend/src/pages/cabins/index.tsx

```diff
@@ -126,9 +155,18 @@ const CabinsPage: NextPage = () => {
       <Grid xs={12} sm={6} item container justifyContent="center">
         {BOOKING_DISABLED ? null : (
           <Link href="/cabins/book" passHref>
-            <Button variant="contained" endIcon={<NavigateNextIcon />}>
-              Book nå
-            </Button>
+            <PermissionRequired permission="cabins.add_booking">
+              <Tooltip open={!isLoggedIn} title="Du må være logget inn for booke en hytte.">
+                <Button
+                  variant="contained"
+                  endIcon={<NavigateNextIcon />}
+                  disabled={!isLoggedIn}
+                  classes={{ root: classes.button }}
+                >
+                  Book nå
+                </Button>
+              </Tooltip>
+            </PermissionRequired>
```

Comment:
Keep in mind that `<PermissionRequired />` will return `null` in all cases where the user does not have the required permission, which will necessarily be the case if the user is not logged in. Either move the `<Tooltip />` outside `<PermissionRequired />`, or expand the `<PermissionRequired />` component to accept a fallback-component. See DOK-216 for more information.